srcpkgs/wren/template

```diff
@@ -0,0 +1,31 @@
+# Template file for 'wren'
+pkgname=wren
+version=0.3.0
+revision=1
+build_wrksrc=./projects/make
+build_style=gnu-makefile
+#make_build_args=""
+#make_dirs="/var/log/dir 0755 root root"
+short_desc="Small, class-based, concurrent scripting language - library files"
+maintainer="cinerea0 <cinerea0@protonmail.com>"
+license="MIT"
+homepage="http://wren.io"
+distfiles="https://github.com/wren-lang/wren/archive/${version}.tar.gz"
+checksum=c566422b52a18693f57b15ae4c9459604e426ea64eddb5fbf2844d8781aa4eb7
+
+do_install() {
+	vinstall ./lib/libwren.a 0755 /usr/lib/
+	vinstall ./lib/libwren.so 0755 /usr/lib/
+	vlicense ./LICENSE
+}
```

Comment:
Do these makefiles not have install targets? Let the makefile do the installation if possible, and move `vlicense` into `post_install`.

Reply:
Just checked; there is no install target. Disregard.